controllers/database.go

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import "gorm.io/gorm"
+
+type DBHandler struct {
+	db *gorm.DB
+}
+
+func NewDBHandler(db *gorm.DB) *DBHandler {
+	return &DBHandler{
+		db: db,
+	}
+}
```

Comment:
I guess this could go inside utils/database itself? Didnt find the utility of creating a seperate file for this,

Reply:
It has to be in the controllers package to define methods on it.

Reply:
in that case renaming dbHandler to soemthing like App should be better to avoid the confusions.